tests/soap/test_soap.py

```diff
@@ -130,7 +130,7 @@ def fake_func(arg: str):
     )
 
 
-def test__retrieve_data_scalar_response(mocker, connector):
+def test__retrieve_data_int_response(mocker, connector):
```

Comment:
Is there any reason for the double underscore prefix in the test signature ?

Reply:
yes, tested function is `_retrieve_data`

Reply:
Ah right, sorry :D